tests/unit/test_session.py

```diff
@@ -27,6 +27,9 @@
 
 
 class ModelA(BaseModel):
+    class Meta:
+        alias = "TransactionModelA"
```

Comment:
Is it needed if not (apparently) used?

Reply:
This is a workaround needed to avoid naming collision with `ModelA` under `tests/unit/test_model.py` (both are defined at the top of the test files before the fixture kicks in) when we discover/trigger tests altogether (it is not a problem triggering the test class on its own).

Another option would be to rename this class to something other than `ModelA`, which would require me to refactor 130+ occurrences in this file, which I thought it wasn't worth it at this point :man_shrugging:.